### PR TITLE
fix(documentation): basic style

### PR DESCRIPTION
**Pull request by rachelbt**
basic side nav style until we will have side nav
`<code>` tag styling
`table` styling

### CODE REVIEW

File: apps/docs/_includes/navigation.njk [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-          <a href="{{ ('/components/' + item.title) | url }}">{{ item.title }}</a>
+          <p><a href="{{ ('/components/' + item.title) | url }}">{{ item.title }}</a></p>
       </li>
       {% endfor %}
     </ul>
   <li>
     </ul>
-{% endblock %}
\ No newline at end of file
+    </nav>
```
Comment (yinonov) [inline apps/docs/_includes/navigation.njk:30]:
```suggestion
</nav>
```

File: apps/docs/assets/styles/main.css [hunk 2]
```diff
@@ -25,3 +25,46 @@ vwc-side-drawer {
...
+  margin: 0;
+}
+.navigation p {
+  color: var(--vvd-color-neutral-70);
+}
+.navigation p a,
+.navigation p a:hover,
+.navigation p a:active{
+  color: var(--vvd-color-neutral-90);
+}
```
Comment (yinonov) [inline apps/docs/assets/styles/main.css:69]: can u region that part with a _temp_ comment?
```css
// #region something...
```

**Review by yinonov — CHANGES_REQUESTED** [review]
add indentation to nested links?

**Comment by rachelbt** [comment]
> add indentation to nested links?

I'll add a little indentation for the list like in the design:
image
Not styling the inner list though cuz currently we dont have nested. I do hope that the UL CSS will be temporary and be replaced with the Side nav :)

**Review by yinonov — APPROVED** [review]

**Pull request merged**